### PR TITLE
# [MO] Unmount a component on page change *(~10 min)*

### PR DESCRIPTION
**Pull request by Nhacsam**

### CODE REVIEW

**Comment by bam-bot-tech** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/page-toggling-feature/react-native/react-navigation/unmount-compoenent-on-page-change.mo.md">unmount-compoenent-on-page-change.mo.md</a>**: Seems you do not need a *Troubleshoot* part
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: react-native/react-navigation/unmount-compoenent-on-page-change.mo.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+ ```jsx
+return ( 
+  <View>
+    {this.props.cameraIsActivated && <Camera />}
+  </View>
+);
+
+ ```
+
+- Test is your component still work (nothing should have change)
```
Comment (tychota) [inline react-native/react-navigation/unmount-compoenent-on-page-change.mo.md:32]: Can you format the checks like this https://github.com/bamlab/dev-standards/blame/master/backend/graphql-js/getting-started-with-apollo-server-dataloader-knex.mo.md#L129 ?

File: react-native/react-navigation/unmount-compoenent-on-page-change.mo.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    navigationOptions: {
+      headerStyle: styles.header,
+      headerTintColor: theme.headerTextColor,
+    },
+    headerMode: 'screen',
+  }
+);
+```
+
+- Test a last time, your component should disapear at the begining of the page transition.
```
Comment (tychota) [inline react-native/react-navigation/unmount-compoenent-on-page-change.mo.md:81]: Idem.

**Review by tychota — APPROVED** [review]
LGTM,

maybe split the steps in two parts and add check format like linked.

**Pull request merged**